.github/workflows/azure_e2e_test.yml

```diff
@@ -0,0 +1,128 @@
+  params-gen:
+    runs-on: ubuntu-latest
+    permissions:
+      contents: 'read'
+      id-token: 'write'
+    outputs:
+      project-name: ${{ steps.gen.outputs.project_name }}
+    steps:
+      - name: Generate a random billing project name
+        id: 'gen'
+        run: |
+          project_name=$(echo "tmp-billing-project-$(uuidgen)" | cut -c -30)
+          echo "project_name=${project_name}" >> $GITHUB_OUTPUT
+
+  create-bee-workflow:
+    runs-on: ubuntu-latest
+    needs: [init-github-context, params-gen]
+    permissions:
+      contents: 'read'
+      id-token: 'write'
+    steps:
+      - name: Dispatch to terra-github-workflows
+        uses: broadinstitute/workflow-dispatch@v3
+        with:
+          workflow: bee-create
+          repo: broadinstitute/terra-github-workflows
+          ref: refs/heads/main
+          token: ${{ env.BROADBOT_TOKEN }}
+          inputs: '{ "bee-name": "${{ env.BEE_NAME }}", "version-template": "dev" }'
+
+  create-and-attach-billing-project-to-landing-zone-workflow:
+    runs-on: ubuntu-latest
+    needs: [ create-bee-workflow, params-gen ]
+    steps:
+      - name: dispatch to terra-github-workflows
+        uses: broadinstitute/workflow-dispatch@v3
+        with:
+          workflow: attach-billing-project-to-landing-zone.yaml
+          repo: broadinstitute/terra-github-workflows
+          ref: refs/heads/main
+          token: ${{ env.BROADBOT_TOKEN }}
+          inputs: '{
+              "run-name": "attach-billing-project-to-landing-zone-${{ env.RUN_NAME_SUFFIX }}",
+              "bee-name": "${{ env.BEE_NAME }}",
+              "billing-project": "${{ needs.params-gen.outputs.project-name }}",
+              "billing-project-creator": "${{ inputs.owner-subject }}",
+              "service-account": "${{inputs.service-account}}" }'
+
+  run-workflows-azure-e2e-test:
+    runs-on: ubuntu-latest
+    needs: [ params-gen, create-and-attach-billing-project-to-landing-zone-workflow ]
+    permissions:
+      contents: 'read'
+      id-token: 'write'
+    steps:
+      - uses: 'actions/checkout@v3'
+      - name: Generate OAuth2 2.0 access token for owner
+        id: 'owner_auth'
+        uses: google-github-actions/auth@v1
+        with:
+          token_format: 'access_token'
+          workload_identity_provider: 'projects/1038484894585/locations/global/workloadIdentityPools/github-wi-pool/providers/github-wi-provider'
+          service_account: ${{ inputs.service-account }}
+          access_token_scopes: 'profile, email, openid'
+          access_token_subject: ${{ inputs.owner-subject }}
+          export_environment_variables: false
+          create_credentials_file: false
+      - uses: actions/checkout@v3
+        with:
+          #NOTE: replace with ${{inputs.target-branch}} before merge
+          #TODO: this might change of the python script is moved to dsp-reusable-workflows
+          ref: refs/heads/sps_azure_e2e_test
```

Comment:
This is so that the action can run the python script that is still in [my branch ](https://github.com/DataBiosphere/cbas/compare/sps_azure_e2e_test)